addons/mrp/models/mrp_workorder.py

```diff
@@ -787,6 +789,13 @@ class MrpWorkorderLine(models.Model):
     finished_workorder_id = fields.Many2one('mrp.workorder', 'Finished Product for Workorder',
         ondelete='cascade')
 
+    @api.onchange('qty_to_consume')
+    def _onchange_qty_to_consume(self):
```

Comment:
i don't get why we need this onchange?

Reply:
it comes from the tests https://drive.google.com/file/d/1WUuuQQmnrCwr63dyCiAlfkujA6use2kC/view